tools/kompos/src/tracingData.ts

```diff
@@ -0,0 +1,110 @@
+/* jshint -W097 */
+'use strict';
+
+import {IdMap, MessageHistoryMessage, MessageData, FarRefData} from './messages';
+import {dbgLog} from './source';
+
+var horizontalDistance = 100,
+    verticalDistance = 100;
+
+export class dataStore {
+    constructor() {
+    }
+
+    private actors = {};
+    private actorsPerType = {};
+    private messages = {};
```

Comment:
Do we know the types for those fields? If yes, would be good if they could be used as type annotations.